src/components/frame/Extensions/pages/ExtensionDetailsPage/ExtensionDetailsPage.tsx

```diff
@@ -33,6 +33,8 @@ import { getValidationSchema } from './validation.ts';
 const displayName = 'frame.Extensions.pages.ExtensionDetailsPage';
 
 const ExtensionDetailsPage: FC = () => {
+  // @TODO replace this with a proper use of context
+  const [activeTab, setActiveTab] = useState(0);
```

Comment:
I'm wondering if this can benefit with a numeric enum so like:

```ts
enum ExtensionDetailsTab {
  Hi,
  Ioanna
}

const [activeTab, setActiveTab] = useState<ExtensionDetailsTab>(ExtensionDetailsTab.Hi);
```

From what I can remember, the library we use to handle tabs accepts numbers to control the tab index or something, hopefully numeric enums don't break that 👀

Reply:
@rumzledz we would definitely benefit from that but I didn't want to overlap too much with Joanna's work here https://github.com/JoinColony/colonyCDapp/blob/5c654be8274ffedc7e1be8f85f08ec4d79383a45/src/components/frame/Extensions/pages/Extensions/ExtensionDetailsPage/types.ts